snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: gazebo
-version: '0.1.0'
+version: '0.1.1'
```

Comment:
why bumping to 0.1.1?

Reply:
because I already tagged 0.1.0 and realized tags were not in the CI files

Reply:
but I think that a change in the CI file should not affect the version of the snap?

Reply:
This versioning being artificial, I don't think it matter so much. Plus, this way we don't remove the history by removing a tag and re-adding it.